### PR TITLE
Add CVE-2015-1840 for jquery-ujs and modify advisory for jquery-rails

### PR DESCRIPTION
**Pull request by reedloden**

### CODE REVIEW

File: gems/jquery-rails/CVE-2015-1840.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
 ---
 gem: jquery-rails
 cve: 2015-1840
-url: https://groups.google.com/forum/#!topic/ruby-security-ann/XIZPbobuwaY
-title: |
-  CSRF Vulnerability in jquery-ujs and jquery-rails
+url: http://blog.honeybadger.io/understanding-the-rails-jquery-csrf-vulnerability-cve-2015-1840/
```
Comment (phillmv) [inline gems/jquery-rails/CVE-2015-1840.yml]: OK, so - url I feel should refer to canonical upstream disclosure.

I definitely like the idea of linking explanatory posts, though. Can you think of an acceptable schema addition?

Comment (reedloden) [inline gems/jquery-rails/CVE-2015-1840.yml]: Could just support multiple urls, with the first one treated as the authoritative. Simple but useful schema change.

Comment (reedloden) [inline gems/jquery-rails/CVE-2015-1840.yml]: As in:
```
url:
  - http://foo.bar
  - http://baz.blah
```

Comment (reedloden) [inline gems/jquery-rails/CVE-2015-1840.yml]: Otherwise, a separate list of `references` or `related_urls` would work.

Comment (reedloden) [inline gems/jquery-rails/CVE-2015-1840.yml]: To unblock this, I just added the URL to the description and swapped back to the old URL.

**Comment by phillmv** [comment]
unfortunately comments get hidden when diffs change, so you wrote 

>Otherwise, a separate list of `references` or `related_urls` would work.

That is exactly what I had in mind. I think references as an array type would be good. For now let's merge this in and create an issue for that.

**Pull request merged**